extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -83,7 +83,7 @@ export class BuildHelper {
 	public async ensureScriptDomDllPresence(outputChannel: vscode.OutputChannel): Promise<boolean> {
 		const scriptdomNugetPkgName = 'Microsoft.SqlServer.TransactSql.ScriptDom';
 		const scriptDomDll = 'Microsoft.SqlServer.TransactSql.ScriptDom.dll';
-		const scriptDomNugetVersion = '161.8817.2'; // TODO: make this a configurable setting, like the Microsoft.Build.Sql version
+		const scriptDomNugetVersion = '161.8910.0'; // TODO: make this a configurable setting, like the Microsoft.Build.Sql version
```

Comment:
the default `microsoftBuildSqlDefaultVersion` above on line 59 should probably also be updated to the latest, 0.1.14-preview

Reply:
and also in the SqlProjects nuget, if that'll be updated anytime soon, since that's what's used when creating new SDK-style projects in ADS and vscode.

Reply:
So, the ScriptDom and Build.Sql can be bumped independently if no dependency exists during the version releases right!

"SqlProjects nuget" means in the STS or in ADS?

Reply:
yup they can be bumped independently. 
The code for the SqlProjects repo is in ADO, and then the projects nuget is specified in STS here: https://github.com/microsoft/sqltoolsservice/blob/main/Packages.props#L32

Reply:
Thanks @kisantia , seems the SqlProjects Nuget version is also needed version update. Will open a PR on STS. Thanks